### PR TITLE
feat: add navbar and po-notes page

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
work done in sprint-1: add navbar, po-notes header and body

### CODE REVIEW

File: src/components/Cards/CardLayout.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import CustomCard from './CustomCard';
+
+const actionItems = [
```
Comment (Kiranyadav1234) [inline src/components/Cards/CardLayout.js:4]: If it's dummy data, please keep it separate and use it here. You can have one folder inside which all the dummy data will be there.

File: src/components/elements/navBar.js [hunk 2]
```diff
@@ -26,15 +20,16 @@ const Navbar = () => {
...
   const handleCloseNavMenu = () => {
     setAnchorElNav(null);
   };
-
   const handleCloseUserMenu = () => {
     setAnchorElUser(null);
   };
 
+  const location = useLocation();
+  console.log(location.pathname);
```
Comment (Kiranyadav1234) [inline src/components/elements/navBar.js:31]: Remove the console. The console is never left in the file.

File: src/components/routes/poNotes.js [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
-        <div>
-            <Navbar/>
-            <PoNotesHeader/>
-            <PoNotesBody/>
-        </div>
-    );
+function PoNotesContainer() {
+  return (
+    <Box>
+      <Box> <PoNotesHeader /> </Box> <br /> <br /> <br />
```
Comment (Harshit-Soni-Mck) [inline src/components/routes/poNotes.js:11]: try not to use line breaks for seperations it is not recommended use styling (padding/margins)

**Comment by samim-hossain-mondal** [comment]
Screenshot 2023-01-31 at 4 23 35 PM
Screenshot 2023-01-31 at 4 23 56 PM

**Comment by samim-hossain-mondal** [comment]
Attaching the snapshots for reference. Please find.

**Pull request closed without merging**